src/Discord.Net.Core/GatewayIntents.cs

```diff
@@ -0,0 +1,39 @@
+using System;
+
+namespace Discord
+{
+    [Flags]
+    public enum GatewayIntents
+    {
+        /// <summary> GUILD_CREATE, GUILD_UPDATE, GUILD_DELETE, GUILD_ROLE_CREATE, GUILD_ROLE_UPDATE, GUILD_ROLE_DELETE, CHANNEL_CREATE, CHANNEL_UPDATE, CHANNEL_DELETE, CHANNEL_PINS_UPDATE </summary>
+        GUILDS = 1 << 0,
```

Comment:
Change all these to PascalCase since that's how it's done everywhere else in the library.
It's also how Microsoft suggests in the [enum naming conventions](https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/names-of-classes-structs-and-interfaces#naming-enumerations).

Reply:
Yep slipped my mind from testing, I copied the names over straight from the docs :)